apps/daimo-web/src/components/DownloadLink.tsx

```diff
@@ -0,0 +1,48 @@
+"use client";
+
+import Link from "next/link";
+import { useEffect, useState } from "react";
+
+export function DownloadLink() {
+  const [, link] = useDownloadTitleLink();
+  return (
+    <Link
+      href={link}
+      target="_blank"
+      className="text-primaryLight font-semibold text-sm"
+    >
+      Download
+    </Link>
+  );
+}
+
+export function DownloadLinkButton() {
+  const [title, link] = useDownloadTitleLink();
+  return (
+    <Link
+      href={link}
+      target="_blank"
+      className="inline-block rounded-lg py-7 px-9 bg-primaryLight text-white font-semibold md:text-xl tracking-wider"
+    >
+      {title}
+    </Link>
+  );
+}
+
+function useDownloadTitleLink() {
+  const [[title, link], setTitleLink] = useState([
+    "Download on App Store",
+    "https://testflight.apple.com/join/j3ixWtuN",
```

Comment:
do we want to start linking the testflight/play store directly already? can link https://noteforms.com/forms/daimo-uk2fe4?notionforms=1&utm_source=notionforms in the short term and switch when we're mainnet-ready